lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
@spovich @Sysqa guys, why don't you just look into the docs before inventing one more bicycle? There's a `Mail#mime_type` method that returns `#{main_type}/#{sub_type}`. It should be as easy as `mail.mime_type == 'text/html'`
http://www.rubydoc.info/github/mikel/mail/Mail/Message#mime_type-instance_method